lib/Elastica/Index.php

```diff
@@ -548,24 +548,24 @@ public function requestEndpoint(AbstractEndpoint $endpoint)
      *
      * Detailed arguments can be found here in the link
      *
-     * @param string $text String to be analyzed
+     * @param string $body String to be analyzed
      * @param array  $args OPTIONAL Additional arguments
      *
      * @return array Server response
      *
      * @link https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-analyze.html
      */
-    public function analyze($text, $args = [])
+    public function analyze($body, $args = [])
     {
         $endpoint = new Analyze();
```

Comment:
I wonder if we should return an error in case body is not a map to make it easier for app devs to detect the problem early.